File: docs/components_page/components/input/textarea.jl
using DashBootstrapComponents, DashHtmlComponents

textareas = html_div([
    dbc_textarea(className = "mb-3", placeholder = "A Textarea"),
    dbc_textarea(
        valid = true,
        size = "sm",
        className = "mb-3",
        placeholder = "A small, valid Textarea",
    ),
    dbc_textarea(invalid = true, size = "lg", placeholder = "A large, invalid Textarea"),
]);

File: docs/components_page/components/input/textarea.R
library(dashBootstrapComponents)

textareas <- htmlDiv(
  list(
    dbcTextarea(className = "mb-3", placeholder = "A Textarea"),
    dbcTextarea(
      valid = TRUE,
      size = "sm",
      className = "mb-3",
      placeholder = "A small, valid Textarea",
    ),
    dbcTextarea(
      invalid = TRUE, size = "lg",
      placeholder = "A large, invalid Textarea"
    )
  )
)
